templates/definition/meter/huawei-emma.yaml

```diff
@@ -0,0 +1,126 @@
+template: huawei-emma
+products:
+  - brand: Huawei
+    description:
+      generic: EMMA
+params:
+  - name: usage
+    choice: ["grid", "pv", "battery"]
+    allinone: true
+  - name: storageunit
+    type: number
+    default: 1
+    advanced: true
+  - name: modbus
+    choice: ["tcpip"]
+  - name: timeout
+    default: 15s
+  - name: capacity
+    advanced: true
+render: |
+  type: custom
+  {{- if eq .usage "grid" }}
+  power:
+    source: modbus
+    id: 0
+    uri: {{ .host }}:{{ .port }}
+    timeout: {{ .timeout }}
+    connectdelay: 1s
```

Comment:
Is this needed?

Reply:
Wie meinst du das? Bzgl. des Templates oder speziefischer?

Reply:
Das Delay?

Reply:
Ich habe das Smartlogger Template als Basis genommen und die Modbus Adressen angepasst. Weiter habe ich nichts hinterfragt. Ich kanns Testen ob es auch ohne das Delay funktioniert.
Was bewirkt dieses genau?

Reply:
Ein unnützes Delay, bitte raus damit.